roles/ceph-validate/tasks/main.yml

```diff
@@ -136,3 +136,10 @@
         msg: "you must add at least one node in the [grafana-server] hosts group"
       when: groups[grafana_server_group_name] | length < 1
   when: dashboard_enabled | bool
+
+- name: validate container registry credentials
+  fail:
+    msg: 'ceph_docker_registry_username and/or ceph_docker_registry_password variables need to be set'
+  when:
+    - ceph_docker_registry_auth | bool
+    - ceph_docker_registry_username is not defined or ceph_docker_registry_password is not defined
```

Comment:
I guess here we might look at if they are empty as well?

Reply:
I guess it wouldn't hurt to check if these variables are empty although I don't really see why you would define them with an empty string.
From a consistency point of view, It would mean we would have to start checking all string variables aren't empty in the whole playbook in that case.

Reply:
but in this specific case, where those variables represent a user/pass set, empty string would cause the login task to fail despite the validation check passing